plugins/zip-extractor/src/index.ts

```diff
@@ -28,8 +28,9 @@ export const ZipExtractor = () => {
             progress: 50,
             info: 'Uploading files',
           })
-          zipEntries.forEach(async (zipEntry) => {
+          const uploadPromises = zipEntries.map(async (zipEntry) => {
             if (
+              !zipEntry.entryName.startsWith('__MACOSX') &&
               !zipEntry.entryName.startsWith('.') &&
               !zipEntry.entryName.match('.DS_Store') &&
```

Comment:
❔ How do these 2 not overlap?

Reply:
@hansjhoffman apparently `entryName` contains the entire path.